bolt/lib/Core/BinaryEmitter.cpp

```diff
@@ -94,6 +100,30 @@ size_t padFunction(const BinaryFunction &Function) {
   return 0;
 }
 
+size_t padFunctionBefore(const BinaryFunction &Function) {
```

Comment:
How similar is the code to `padFunction()`? Can you reduce the code duplication?

Reply:
Done.